.github/workflows/ci-go.yaml

```diff
@@ -0,0 +1,22 @@
+name: CI (Go)
+
+on:
+  push:
+  workflow_dispatch:
+
+concurrency:
+  group: ci-go-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+    name: Build Runner
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v2.3.4
+
+      - name: Build Runner
+        run: make
+        working-directory: runner
```

Comment:
Oh, we can actually scrap any runner-related stuff in this. It's being moved to https://github.com/Automattic/cron-control-runner and the build process will be different than seen here.

Reply:
@WPprodigy just to confirm, can I remove the entire `runner` directory?

Reply:
Not yet, I have a deprecation plan. But just don't worry about any improvements or building.

Reply:
OK

Reply:
Done